pkg/solana/logpoller/parser.go

```diff
@@ -21,6 +21,7 @@ const (
 	txHashFieldName       = "tx_hash"
 	addressFieldName      = "address"
 	eventSigFieldName     = "event_sig"
+	eventSubkeyFieldName  = "event_subkey"
```

Comment:
This file needs some more cleaning up. It looks like all of the changes to this file (and to `parser_test.go` I think?) should be reverted, aside from the addition of the `FormatContractReaderCursor(log Log) string` function.

For example, there is no `event_subkey` column in the logpoller tables, it's `subkey_values` which is already defined. The `eventSubkeyFilter` and all of its methods should be removed, we just need `eventBySubKeyFilter`.

Reply:
What is the `SubKeyIndex`? How do I map a subkey path to an index?